navigator_engine/api.py

```diff
@@ -77,3 +77,34 @@ def action(action_id):
         'id': action_id,
         'content': action.to_dict()
     })
+
+
+@api_blueprint.route('/actionlist', methods=['POST'])
```

Comment:
how about `/action/list` endpoint?

Reply:
Suggests to me it is a property of an action? Get me the list belonging to action...

We could do `/decide` `/decide/list` `/decide/action`?